src/index.js

```diff
@@ -140,6 +140,80 @@
                             });
                         });
                 },
+                putUser (username, metadata = {}, db = databases[ defaultDB ]) {
+                    return db
+                        .putUser(username, {
+                            metadata
+                        })
+                        .then(() => {
```

Comment:
Not sure why you want to trigger the same instance function again when the promise is resolved. This is a double call in my opinion

Reply:
you're right, I don't know why, coding too fastly.